modules/optimization/test/tests/optimizationreporter/general_opt/point_loads/gold/main_transfer_data_out.csv

```diff
@@ -0,0 +1,3 @@
+time,OptimizationReporter/misfit_norm
```

Comment:
I don't think we need to check the objective value, it should be 0.  Just getting the parameter_results correct is good enough.
You can add outputs=none in the block that is creating this, if that will work.